VotingApplication/VotingApplication.Web.Api/Scripts/BasicVote.js

```diff
@@ -152,47 +83,62 @@
 
         self.displayResults = function (data) {
             var groupedVotes = countVotes(data);
-            drawChart(groupedVotes);
-        }
+            self.drawChart(groupedVotes);
+        };
+
+        self.initialise = function (pollData) {
+            self.pollOptions.initialise(pollData);
+        };
 
-        self.addOption = function () {
-            //Don't submit without an entry in the name field
-            if ($("#newName").val() === "") {
+        // TODO: Extract chart code from viewModel class - ideally
```

Comment:
It's what I'm planning to do next under VA-183 - Remove view code from view model
https://int.scottlogic.co.uk/jira/browse/VA-183